cmd/preflight/cmd/helper_test.go

```diff
@@ -326,6 +326,13 @@ var _ = Describe("Preflight cmd helper unit tests", func() {
 		})
 	})
 
+	Context("validateInitKubeEnv func test-case", func() {
+		It("Should perform preflight check using kubeconfig file mentioned in KUBECONFIG env", func() {
+			preflight.InitKubeEnv("")
+			Expect(kubeconfig).Should(Equal(os.Getenv(internal.KubeconfigEnv)))
```

Comment:
Why do you need to call the `initKubeEnv`  func here?

Here we want to check the value of kubeconfig flag, and not the flag string(`--kubeconfig`)
You should check whether the `cmdOps.Kubeconfig` field value has been initialized properly.